spec/controllers/objects_controller_spec.rb

```diff
@@ -5,15 +5,22 @@
   let(:prefixed_druid) { 'druid:bj102hs9687' }
   let(:prefixed_druid2) { 'druid:bz514sm9647' }
   let(:bare_druid) { 'bj102hs9687' }
+  let(:bare_druid2) { 'bz514sm9647' }
```

Comment:
it was lucky (for reducing churn in this PR) that `bare_druid` already paralleled `prefixed_druid`, and that `prefixed_druid` and `prefixed_druid2` were already in alphabetical order.